From 42aebb28332e0bdae20b449c6de19d9e363c40a3 Mon Sep 17 00:00:00 2001
From: Joe Perches <joe@perches.com>
Date: Sun, 29 Jan 2012 13:47:52 +0000
Subject: drivers/net: Remove alloc_etherdev error messages

alloc_etherdev has a generic OOM/unable to alloc message.
Remove the duplicative messages after alloc_etherdev calls.

Signed-off-by: Joe Perches <joe@perches.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 41de8d4cff21a2e81e3d9ff66f5f7c903f9c3ab1)

Conflicts:

	drivers/net/ethernet/3com/3c59x.c
	drivers/net/ethernet/adaptec/starfire.c
	drivers/net/ethernet/adi/bfin_mac.c
	drivers/net/ethernet/alteon/acenic.c
	drivers/net/ethernet/amd/amd8111e.c
	drivers/net/ethernet/amd/au1000_eth.c
	drivers/net/ethernet/amd/declance.c
	drivers/net/ethernet/amd/pcnet32.c
	drivers/net/ethernet/apple/bmac.c
	drivers/net/ethernet/apple/mace.c
	drivers/net/ethernet/atheros/atl1c/atl1c_main.c
	drivers/net/ethernet/atheros/atl1e/atl1e_main.c
	drivers/net/ethernet/broadcom/b44.c
	drivers/net/ethernet/broadcom/bnx2x/bnx2x_main.c
	drivers/net/ethernet/broadcom/sb1250-mac.c
	drivers/net/ethernet/broadcom/tg3.c
	drivers/net/ethernet/brocade/bna/bnad.c
	drivers/net/ethernet/cadence/macb.c
	drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
	drivers/net/ethernet/cisco/enic/enic_main.c
	drivers/net/ethernet/davicom/dm9000.c
	drivers/net/ethernet/dec/tulip/tulip_core.c
	drivers/net/ethernet/dnet.c
	drivers/net/ethernet/ethoc.c
	drivers/net/ethernet/hp/hp100.c
	drivers/net/ethernet/ibm/ehea/ehea_main.c
	drivers/net/ethernet/ibm/emac/core.c
	drivers/net/ethernet/ibm/iseries_veth.c
	drivers/net/ethernet/icplus/ipg.c
	drivers/net/ethernet/intel/e100.c
	drivers/net/ethernet/jme.c
	drivers/net/ethernet/korina.c
	drivers/net/ethernet/lantiq_etop.c
	drivers/net/ethernet/marvell/skge.c
	drivers/net/ethernet/marvell/sky2.c
	drivers/net/ethernet/mellanox/mlx4/en_netdev.c
	drivers/net/ethernet/micrel/ks8695net.c
	drivers/net/ethernet/micrel/ks8851.c
	drivers/net/ethernet/microchip/enc28j60.c
	drivers/net/ethernet/myricom/myri10ge/myri10ge.c
	drivers/net/ethernet/neterion/s2io.c
	drivers/net/ethernet/netx-eth.c
	drivers/net/ethernet/oki-semi/pch_gbe/pch_gbe_main.c
	drivers/net/ethernet/packetengines/yellowfin.c
	drivers/net/ethernet/pasemi/pasemi_mac.c
	drivers/net/ethernet/qlogic/netxen/netxen_nic_main.c
	drivers/net/ethernet/qlogic/qla3xxx.c
	drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
	drivers/net/ethernet/rdc/r6040.c
	drivers/net/ethernet/realtek/8139too.c
	drivers/net/ethernet/realtek/r8169.c
	drivers/net/ethernet/s6gmac.c
	drivers/net/ethernet/seeq/sgiseeq.c
	drivers/net/ethernet/sis/sis190.c
	drivers/net/ethernet/smsc/epic100.c
	drivers/net/ethernet/smsc/smc911x.c
	drivers/net/ethernet/smsc/smc91x.c
	drivers/net/ethernet/smsc/smsc911x.c
	drivers/net/ethernet/smsc/smsc9420.c
	drivers/net/ethernet/stmicro/stmmac/stmmac_main.c
	drivers/net/ethernet/sun/cassini.c
	drivers/net/ethernet/sun/niu.c
	drivers/net/ethernet/sun/sungem.c
	drivers/net/ethernet/sun/sunvnet.c
	drivers/net/ethernet/tehuti/tehuti.c
	drivers/net/ethernet/ti/cpmac.c
	drivers/net/ethernet/ti/davinci_emac.c
	drivers/net/ethernet/ti/tlan.c
	drivers/net/ethernet/toshiba/tc35815.c
	drivers/net/ethernet/tundra/tsi108_eth.c
	drivers/net/ethernet/via/via-rhine.c
	drivers/net/ethernet/via/via-velocity.c
	drivers/net/ethernet/xilinx/ll_temac_main.c
	drivers/net/ethernet/xilinx/xilinx_axienet_main.c
	drivers/net/ethernet/xilinx/xilinx_emaclite.c
	drivers/net/plip/plip.c
	drivers/net/rionet.c
	drivers/net/tulip/xircom_cb.c
	drivers/net/usb/pegasus.c
	drivers/net/usb/rtl8150.c
	drivers/net/usb/usbnet.c
	drivers/net/vmxnet3/vmxnet3_drv.c
	drivers/net/wireless/atmel.c
	drivers/net/wireless/ipw2x00/libipw_module.c
	drivers/net/xen-netfront.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 drivers/net/ethernet/renesas/sh_eth.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/renesas/sh_eth.c b/drivers/net/ethernet/renesas/sh_eth.c
index 813d41c..1cb5a34 100644
--- a/drivers/net/ethernet/renesas/sh_eth.c
+++ b/drivers/net/ethernet/renesas/sh_eth.c
@@ -1792,7 +1792,6 @@ static int sh_eth_drv_probe(struct platform_device *pdev)
 
 	ndev = alloc_etherdev(sizeof(struct sh_eth_private));
 	if (!ndev) {
-		dev_err(&pdev->dev, "Could not allocate device.\n");
 		ret = -ENOMEM;
 		goto out;
 	}
-- 
1.7.10

